### DISCUSSION TITLE
Unable to create

### DISCUSSION THREAD

**Discussion by ajithk444** [microsoft/dicom-server#553] [Q&A] [answered]
Hello All,

I setup my DICOM server in azure. Now I am not able to create DicomClient object as mentioned in the step in this page.

https://github.com/microsoft/dicom-server/blob/main/docs/tutorials/use-dicom-web-standard-apis-with-c%23.md#create-a-dicomwebclient

please help me on what I am missing. I created a new console application with .NET 5.0 and added below code to the console.

`string webServerUrl ="https://dicomweb.azurewebsites.net/"
var httpClient = new HttpClient();
httpClient.BaseAddress = new Uri(webServerUrl);
IDicomWebClient client = new DicomWebClient(httpClient);`

**Comment by smithago** [comment, marked as answer]
Hi Ajit,

What is the issue you are running into? Did you download the nuget package and reference it in your project? The nuget feed is available at https://microsofthealthoss.visualstudio.com/FhirServer/_packaging?_a=package&feed=Public&package=Microsoft.Health.Dicom.Client&protocolType=NuGet&version=1.0.0-main-20210125-1.

**Reply by ajithk444** [reply]
Hi Smitha,

I was getting DLL load issue as I complied the source code and referenced the using Microsoft.Health.Dicom.Client; locally. it did not work. Thanks for sharing this package details and I am now able to create the DicomWebClient without any issues.

Is there any sample app in c# that makes all the API calls in C#.

**Reply by abdullah248** [reply]
Hi Ajith,
The original [tutorial ](https://github.com/microsoft/dicom-server/blob/main/docs/tutorials/use-dicom-web-standard-apis-with-c%23.md) you were following goes through all of the available API calls using the DicomWebClient.
